core/templates/dev/head/domain/topic/TopicObjectFactory.js

```diff
@@ -285,10 +285,12 @@ oppia.factory('TopicObjectFactory', ['SubtopicObjectFactory',
       return new Topic(topicBackendObject);
     };
 
-    // Create a new, empty topic. This is not guaranteed to pass validation
-    // tests.
-    Topic.createEmptyTopic = function() {
+    // Create an interstitial topic that would be displayed in the editor until
+    // the actual topic is fetched from the backend.
+    Topic.createInterstitialTopic = function() {
       return new Topic({
```

Comment:
Sorry I missed this earlier: could you make the topic constructor have individual args, same as for TopicRights?

Reply:
Done.